diff --git a/feeds/luci/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js b/feeds/luci/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js
--- a/feeds/luci/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js
+++ b/feeds/luci/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js
@@ -86,7 +86,7 @@ return view.extend({
 			o = s.option(form.Flag, 'flow_offloading',
 				_('Software flow offloading'),
 				_('Software based offloading for routing/NAT'));
-			o.optional = true;
+			o.optional = false;
 
 			o = s.option(form.Flag, 'flow_offloading_hw',
 				_('Hardware flow offloading'),
@@ -94,7 +94,14 @@ return view.extend({
 			o.optional = true;
 			o.depends('flow_offloading', '1');
 		}
-
+		/* NATflow flow offloading support */
+		/* if (L.hasSystemFeature('natflow')) { */
+			o = s.option(form.Flag, 'natflow',
+			_('NATflow flow offloading'),
+			_('A fast forwarding stanalone kernel module'));
+			o.optional = false;
+			o.depends('flow_offloading', '0');
+		/* 	} */
 
 		s = m.section(form.GridSection, 'zone', _('Zones'));
 		s.addremove = true;
